### PR TITLE
Benny/remove nft table

### PR DESCRIPTION
**Pull request by bennycio**
Changes:

- **Update** docker files to reflect migrations
- **Removed** NFT table and removed any reference to the NFT table in persist.

### CODE REVIEW

File: db/migrations/000011_nft_removal.down..sql [hunk 1]
```diff
@@ -0,0 +1,30 @@
+
```
Comment (radazen) [inline db/migrations/000011_nft_removal.down..sql:1]: Looks like there's an extra dot in this filename!

**Review by radazen — APPROVED** [review]
Left two comments, but LGTM aside from a quick filename fix! 🔥

File: service/persist/token.go [hunk 2]
```diff
@@ -203,6 +203,65 @@ type Media struct {
 	MediaType    MediaType  `json:"media_type"`
 }
 
+// NFT represents an old nft throughout the application
+type NFT struct {
```
[the remark this answers is not in the document]
Comment (bennycio) [reply service/persist/token.go:207]: It is being used in the migration so will just keep it around until we are merged over 👍🏻

File: db/migrations/000011_nft_removal.down..sql (hunk 1 above)
Comment (bennycio) [reply db/migrations/000011_nft_removal.down..sql:1]: lol fixing that now

**Pull request merged**